src/bootstrap.js

```diff
@@ -173,6 +191,12 @@ function startup({webExtension}, reason) {
           message.extra || null
         );
         respond({});
+        break;
+      case "telemetry_scalar":
+        Services.telemetry.scalarSet(
+          TELEMETRY_CATEGORY + message.name, message.value
```

Comment:
I should have called this out in the first review (sorry!):

I think it would be better for the scalar names to be "plain" (e.g., no leading `.`).  To that end, this line should change to something like:

```javascript
`${TELEMETRY_CATEGORY}.${message.name}`
```